finality-aleph/src/lib.rs

```diff
@@ -180,5 +191,8 @@ where
     C::Api: aleph_primitives::AlephSessionApi<B>,
     SC: SelectChain<B> + 'static,
 {
-    run_consensus_party(AlephParams { config })
+    match config.node_type {
```

Comment:
Maybe instead of calling separate functions we could just have `run_node` and it would call something like `run_consensus` inside depending on config? `run_consensus` would spawn stuff like  `network_task`, `network_manager_task`, `handler_task` and party.

Reply:
same thing but pushed deeper? Sounds ok

Reply:
Yep, pretty much

Reply:
Actually, I like how it is right now more. Imho this dispatch should be done at this level

Reply:
This `if intersport` pattern doesn't look good here ;) Perhaps if there would be anything beyond just forwarding that call. That match in `main` should be more than enough and you already divided it, i.e. `new_full` and `new_nonvalidator`.